Sources/Plasma/PubUtilLib/plGImage/plDynamicTextMap.cpp

```diff
@@ -428,15 +428,7 @@ void    plDynamicTextMap::SetFont( const ST::string &face, uint16_t size, uint8_
 //  if( !IIsValid() )
 //      return;
 
-    if (plLocalization::UsingUnicode())
-    {
-        // unicode has a bunch of chars that most fonts don't have, so we override the font choice with one
-        // that will work with the desired language
-        hsStatusMessageF("We are using a unicode language, overriding font choice of %s", face.c_str("nil"));
-        fFontFace = "Unicode";
-    }
-    else
-        fFontFace = face;
+    fFontFace = face;
```

Comment:
This concerns me a little, since we already know that most of the fonts don't include any characters past `U+00FF`...  At some point, we'll probably need to expand the fonts if we want to actually render languages that require other unicode characters.

Reply:
Interestingly, this code appears to have been copied from Myst V (which does have giant Unicode P2F files), while Uru never shipped any Unicode font files

Reply:
Yeah, I agree this isn't optimal, but the code that I removed here is dead and broken in its current state - it would need to be reworked anyway if we ever actually support languages not covered by the existing fonts. I explained my reasoning a bit more in the commit message for 7042d18163c9983f650da4056ec03cabf72266ed.